src/lib/dependencies/nuget.js

```diff
@@ -1,72 +1,26 @@
-import debug from 'debug';
-
-import cache from '../cache';
-
 import xmldoc from 'xmldoc';
 
-import { searchFilesFromRepo } from '../github';
-
-import { flatten } from 'lodash';
-
-const _debug = debug('dependencies:nuget');
+const patterns = {
+  'packages.config': packagesConfigDependencies,
+  '*.csproj': csprojDependencies,
+};
```

Comment:
In that case, we were (on purpose) first looking for `*.csproj` then for `packages.config`. We were not looking for `packages.config` if there are dependencies detected in `*.csproj`. We're doing this that way because GitHub has a strong rate limiting on the search endpoint and we want to spare requests there.

Reply:
Yes, I realized that (see `firstSetOfMatchingFiles` in `index.js`, but I may have switched the order though, I'll check that.)